staging/src/k8s.io/legacy-cloud-providers/openstack/util/mount/mount.go

```diff
@@ -0,0 +1,410 @@
+/*
```

Comment:
FYI for reviewers, we agreed that copying `pkg/util/mount` here is a temporary measure until https://github.com/kubernetes/utils/pull/100 is merged or when we remove the open stack provider entirely targeted for v1.17

Reply:
I'm not eager to have three copies of this code in flight. What's the timeframe for https://github.com/kubernetes/utils/pull/100 to merge?

cc @msau42

Reply:
We are targeting 1.16 but it depends on if @codenrhoden has enough cycles to address Tim's feedback by code freeze

Reply:
@liggitt @msau42 I agree with that sentiment. Code duplication has a certain stink that I'm not fond of. On the other hand, I've been talking about getting this code removed for almost two years now, and I'd like to finally follow through on that promise.

The options would be to:

1. Hold the change until https://github.com/kubernetes/utils/pull/100 lands then make the appropriate changes to use that library
2. Merge the change and if the utils patch merges switch the library over before code freeze if possible.

My preference is 2. In my mind we're making things slightly worse in the short term to make the much better in the long term. The risk is that we adopt this bit of stink then never revisit it, but I'll take responsibility for making sure that doesn't happen.

Reply:
The mount util code has been frequently modified to address functional and security-related issues. It has test coverage in CI, but only because it is used by volume types available in CI environments. Making a copy here would make it very easy to miss propagating bug and security fixes, and would make it very difficult to test those fixes in CI.

Reply:
Sure, if we think the utils patch would land I would strongly prefer we use that. In two weeks remaining before code freeze, do we think that will happen? If so, then I fully support option 1 for the reasons you stated.

One of the reasons we want to make this move a bit more urgently is to support this KEP to build a K8s without a provider.

https://github.com/kubernetes/enhancements/pull/1180